app/src/main/java/com/codelabs/basicstatecodelab/WellnessViewModel.kt

```diff
@@ -0,0 +1,21 @@
+package com.codelabs.basicstatecodelab
+
+import androidx.compose.runtime.toMutableStateList
+import androidx.lifecycle.ViewModel
+
+class WellnessViewModel : ViewModel() {
+    private val _tasks = getWellnessTasks().toMutableStateList()
+    val tasks: List<WellnessTask>
+        get() = _tasks
+
+    fun remove(item: WellnessTask) {
+        _tasks.remove(item)
+    }
+
+    fun changeTaskChecked(item: WellnessTask, checked: Boolean) =
+        tasks.find { it.id == item.id }?.let { task ->
+            task.checked = checked
+        }
```

Comment:
Here you turned `checked` into a mutable state (as the suggested approach in the tutorial). A more-React way of doing that would be to update the whole list. Could you implement that alternative solution?